docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--org-section-long-form--default.html.twig

```diff
@@ -1,22 +1,32 @@
-{% set heading = { } %}
+{% set title = '' %}
 {% if paragraph.field_hide_heading is not defined or paragraph.field_hide_heading.0.value == 0 %}
-  {% set heading = {
-    "title": (content.field_section_long_form_heading|field_value),
-    "id": "",
-    "borderless": true
-  } %}
+  {% set title = content.field_section_long_form_heading|field_value %}
+{% endif %}
+
+{% set contained = false %}
+{% set paragraphType = content.field_section_long_form_content[0]["#paragraph"]["#values"]["type"] %}
```

Comment:
@clairesunstudio the problem is that content.field_section_long_form_content contains 1 or more childs, field_section_long_form_content is each organization section sub paragraph, so by doing this: field_section_long_form_content[0] you are checking just for the first element, but if instead the second or third element is one of those, it will be ignored. In any case you need to add a wrapper to the subchilds, not the main wrapper (paragraph org section).